data_model/src/lib.rs

```diff
@@ -557,6 +557,7 @@ pub enum Value {
     FfiType,
     IntoSchema,
 )]
+#[ffi_type(opaque)]
```

Comment:
troubles with `Option<&NumericValue>`. Opened a ticket to resolve at some point in the future